zipline-gradle-plugin/src/main/kotlin/app/cash/zipline/gradle/ZiplinePlugin.kt

```diff
@@ -80,6 +82,23 @@ class ZiplinePlugin : KotlinCompilerPluginSupportPlugin {
       createdTask.mainModuleId.set(configuration.mainModuleId)
       createdTask.mainFunction.set(configuration.mainFunction)
       createdTask.version.set(configuration.version)
+
+      fun <T> Iterable<Provider<T>>.flatten(): Provider<List<T>> {
```

Comment:
Off topic: sooo much work to support providers

Reply:
It's probably unnecessary here as the hex could be just a string, but if you needed to load your keys with a Gradle task allow you to do so in the proper way. Going to file an issue with Gradle about making this easier.